frontend/test/simulate.run/frontend simulations.run.xml

```diff
@@ -0,0 +1,18 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
What is this?

Reply:
makes goland slightly easier to use for launching tests in debug mode.